dashboard/README.md

```diff
@@ -40,29 +40,38 @@ dashboards with different configurations.
 [replicator]: cmd/postgres_replicator/README.md
 [replicatorconfig]: config/postgres_replicator/default/config.yaml
 
+## Deploy dashboard with PSM
```

Comment:
The five lines above this one should be moved so they appear at the bottom of the section where they are used. There should be no spaces at the beginning of the line.